chunky/src/java/se/llbit/chunky/block/ModelBlock.java

```diff
@@ -11,7 +11,5 @@
   @PluginApi
   BlockModel getModel();
 
-  default boolean intersect(Ray ray, Scene scene) {
-    return getModel().intersect(ray, scene);
-  }
+  boolean intersect(Ray ray, Scene scene);
```

Comment:
Why?

Reply:
The `intersect` in block overrides this `default`.

Reply:
We can just remove the `intersect` method from this interface then. It's unused.